source/plugins/ipv6.rst

```diff
@@ -199,3 +202,117 @@ each of the VMs. Use DUID-LL to set up dhclient.
          iface eth0 inet6 dhcp
          autoconf 0
          accept_ra 1
+
+
+Isolated network and VPC tier
+-----------------------------
+
+.. note::
+   - The IPv6 support for isolated networks and VPC tiers is available from version 4.17.0.
+
+   - The IPv6 isolated networks and VPC tiers only supports **Static routing**, i.e, the administrator will need to add upstream routes for routing to work inside the networks.
+
+Guest VMs in an isolated network or VPC tier can obtain both IPv4 and IPv6 IP addresses by using a supported network offering and appropriate configurations for IPv6 support by the administrator.
+Both VR for such networks and the guest VMs using these networks obtain a SLAAC based IPv6 address. While VR is assigned an IPv6 address from the public IPv6 range, guest VMs get their IPv6 addresses from the IPv6 subnet assinged to the network.
+
+Here's the sequence of events when IPv6 is used:
```

Comment:
Is the "VLAN/VNI" supported in ipv6 range ?
If it is currently not supported, it is better to point it out.

Reply:
@weizhouapache will sync offline about this.